### ISSUE TITLE
Documentation locations

### ISSUE THREAD

**Issue by schloerke** [rstudio/plumber#353]
Currently:
* `gh-pages` branch is a book-like documentation website
* there is no pkgdown form of documentation
* documentation is separated from code by being in gh-pages branch

Proposal:
* Move current `gh-pages` to it's own repo
  * able to track documentation issues separately (and with more liberal frequency)
* Make a `./docs` folder for output of pkgdown
  * delete `gh-pages` branch
  * or keep gh-pages as a place to travis deploy pkgdown and never commit any pkgdown output

@trestletech @blairj09  Thoughts?

**Comment by trestletech** [comment]
That sounds fine to me. I'm not sure how useful pkgdown would be given how little of the docs are in .Rd right now.

The original motivation for this was two-fold:

1. There was a lot of content that I thought needed to be explained that didn't fit nicely into my little RStudio IDE help window.
2. I hadn't found an elegant way to document R6 methods in Roxygen, so I gave up on it pretty quickly and started rolling my own docs. 

Oh, and I guess 3. It'd be neat to turn the bookdown docs into a book at some point. 

But I've got no qualms with what you suggest, though I'm skeptical that the pkgdown site may not have much value.
